docs/testing-components/test-driven-development/index.md

```diff
@@ -9,9 +9,9 @@ the basic information on what unit testing is and how to start unit testing.
 
 ## Overview
 
-This document describes the test-first methodology we aspire to on the Enact team.  The concept behind Test Driven
-Development (TDD) is that you write tests before you begin implementation.  The theory is that testing helps you think
-through the problem and serves as a sort of design for the forthcoming code.  Additionally, the tests serve as a validation
+This document describes the test-first methodology we aspire to on the Enact team. The concept behind Test Driven
```

Comment:
Please do not change double spaces to a single space. It(double spaces) means next line.

Reply:
Done